From: =?utf-8?q?Antoine_Beaupr=C3=A9?= <anarcat@debian.org>
Date: Sat, 10 Aug 2024 12:42:15 -0700
Subject: fix the paths to javascript files

Forwarded: not-needed
Last-Update: 2019-08-25

We're using javascript libraries from debian packages instead of the ones
vendored by upstream. Because of this some paths to libraries might differ a
bit.
Some folks are still using URLs of the form /cgi-bin/smokeping.cgi (which was
used up until buster). In order for css and js to be able to load with this
URL, we need to use our top-level alias to retrieve them.
---
 etc/basepage.html.dist | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/etc/basepage.html.dist b/etc/basepage.html.dist
index 2c0694f..92cfa54 100644
--- a/etc/basepage.html.dist
+++ b/etc/basepage.html.dist
@@ -4,8 +4,8 @@
     <link href="data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1gQSFA8qQC+y8wAAAB10RVh0Q29tbWVudABDcmVhdGVkIHdpdGggVGhlIEdJTVDvZCVuAAABd0lEQVQoz5WSy07CQBSGTzuF6Y1buQSiCYSgYli7MC54AZc+gC/CE7kzrPQBjDsNIdF4I8oCwr1gO51OOy4aAbGS+K/OTM6X85/5R2g0GvAfSesHjPyj3LyasrIKxch3fXHiSN0Fvv4wbCZuAobsnld7Scw4wIhEhiSCEc/IbgqzZicTMuG0NEpi9jhRLzuZOUXBJRK4ITOfr1yIy6oYIwDQ7KSX3QDgcWFgR9ZtrwCLIQAoJ8j2pVG9Xg8qDsJe0j5MWcW4g5FvMUQ8cRvQXeAplXY0mlfpftI+zpu1tBURed/CHhdCAADoWfiml3gxlYUraZKXVdxKwq6lPx8mqvM97QcQaEalV1O57cffTGVXdzKKa8isNdI3l/6tzly+eM4BQDlOQl4pVEECLDSHUowUNCqsdedVelYZAEB7rIUkfVKYHaQs4okTIjEuJKJePMoCY1fvRghwN9R9gIJKs4orCtxm6Gmq3I/01lDnob+1PdbWR/+lL5mtm+fskIeBAAAAAElFTkSuQmCC" rel="icon" type="image/x-icon" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>SmokePing Latency Page for <##title##></title>
-    <link rel="stylesheet" type="text/css" href="css/smokeping-print.css" media="print">
-    <link rel="stylesheet" type="text/css" href="css/smokeping-screen.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="/smokeping/css/smokeping-print.css" media="print">
+    <link rel="stylesheet" type="text/css" href="/smokeping/css/smokeping-screen.css" media="screen">
     <script>
     	window.options = {
 	    step: <##step##>
@@ -56,10 +56,10 @@
     <p><small>Maintained by <a href="mailto:<##contact##>"><##owner##></a></small></p>
 </div>
 
-<script src="js/prototype.js" type="text/javascript"></script>
-<script src="js/scriptaculous/scriptaculous.js?load=builder,effects,dragdrop" type="text/javascript"></script>
-<script src="js/cropper/cropper.js" type="text/javascript"></script>
-<script src="js/smokeping.js" type="text/javascript"></script>
+<script src="/smokeping/js/prototype/prototype.js" type="text/javascript"></script>
+<script src="/smokeping/js/scriptaculous/scriptaculous.js?load=builder,effects,dragdrop" type="text/javascript"></script>
+<script src="/smokeping/js/cropper/cropper.js" type="text/javascript"></script>
+<script src="/smokeping/js/smokeping.js" type="text/javascript"></script>
 
 </body>
 </html>
